eng/WpfArcadeSdk/tools/GenApi.targets

```diff
@@ -4,13 +4,30 @@
                       Condition="'$(_GenerateReferenceAssemblySource)'=='true'"/>
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(_GenerateReferenceAssemblySource)'=='true'">
+  <PropertyGroup Condition="'$(_GenerateReferenceAssemblySource)'=='true'
+                                or $(MSBuildProjectName.Contains('_wpftmp'))">
```

Comment:
I don't think you want to run GenAPI when `$(MSBuildProjectName.Contains('_wpftmp')`, right?